### PR TITLE
versions update kernel to 4.19.52

### PR DESCRIPTION
**Pull request by egernst**
Fixes: #1816

Signed-off-by: Eric Ernst <eric.ernst@intel.com>

### CODE REVIEW

**Comment by egernst** [comment]
@Pennyzct  Can you PTAL and fix-up the ARM patch?

**Comment by Pennyzct** [comment]
Hi~ @egernst should be late for you, so working 12/12/7 like @gnawux 😎🤐
I‘ll try it locally and do the fix-up asap.;)
BTW, you may need to add commit body to pass the static check.😜

File: versions.yaml [hunk 1]
```diff
@@ -164,7 +164,7 @@ assets:
     url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
     uscan-url: >-
       https://mirrors.edge.kernel.org/pub/linux/kernel/v4.x/linux-(4\.19\..+)\.tar\.gz
-    version: "v4.19.28"
+    version: "v4.19.52"
```
Comment (grahamwhaley) [inline versions.yaml]: upstream is at .53 already - just to note. .52 has the CVE fix in it though, so we can roll with this.

**Comment by gnawux** [comment]
@egernst try with @Pennyzct 's patch in kata-containers/packaging#592

**Comment by Pennyzct** [comment]
Hi~ @grahamwhaley 
```
08:30:09 Install chronic
08:30:09 
08:30:09 WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
08:30:09 
08:30:09 E: Could not get lock /var/lib/dpkg/lock-frontend - open (11: Resource temporarily unavailable)
08:30:09 E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?
08:30:09 Failed at 18: sudo -E apt install -y moreutils
```
It again failed at the error which I said in [tests/#1720](https://github.com/kata-containers/tests/pull/1720). I will log into ARM CI node to check if there existed hanging `apt` process and give it back a clean environment, Then I will follow you suggestion to check whether it has auto update services running in the background.

**Comment by Pennyzct** [comment]
I have cleaned up the environment and I will restart ARM CI again. ;)

**Comment by gnawux** [comment]
almost done, could this be backported to stable branches, @bergwolf? (I think you are out of the office today, right?)

**Comment by bergwolf** [comment]
@gnawux I'm leaving in the afternoon. I can backport this once it's merged. Do you think it is ready now?

**Comment by gnawux** [comment]
@bergwolf I think so, all the CIs are passed except the running ARM one and @Pennyzct is actively working on it.

**Comment by gnawux** [comment]
As I commented in kata-containers/packaging#592 ([here](https://github.com/kata-containers/packaging/pull/592#issuecomment-504286327)), the failure in the ARM test is caused by lacking ptp0 device, however, looks the fix in kata-containers/osbuilder#265 won't be merged soon. I'd like to merge this patch and kata-containers/packaging#592 once the required test in 592 is passed.

**Comment by gnawux** [comment]
kata-containers/packaging#592 landed, let's merge this. And @bergwolf, please help backport to stable once you have time.

**Pull request merged**